portal-impl/src/com/liferay/portlet/announcements/service.xml

```diff
@@ -52,8 +52,8 @@
 
```

Comment:
removed `uad-anonymize-field-name="screenName"` on userName since ServiceBuilder generate that automatically